skafka/src/test/scala/com/evolutiongaming/skafka/consumer/ConsumerConfigSpec.scala

```diff
@@ -45,6 +45,18 @@ class ConsumerConfigSpec extends AnyFunSuite with Matchers {
       clientCallbackHandlerClass      = Some(classOf[ConsumerConfigSpec]),
       loginCallbackHandlerClass       = Some(classOf[ConsumerConfigSpec]),
       loginClass                      = Some(classOf[ConsumerConfigSpec]),
+    ),
+    sslSupport = new SslSupportConfig(
```

Comment:
no need for `new`, simple `SslSupportConfig` should work just fine,
same in other places

Reply:
Fixed